tests/functional/model/run_model_test.py

```diff
@@ -94,3 +123,15 @@ def test_llm_run_with_file():
     # Verify response
     assert response["status"] == "SUCCESS"
     assert "🤖" in response["data"], "Robot emoji should be present in the response"
+
+
+
+
+@pytest.mark.parametrize("function, selected_model, input_data, output_keyword", fetch_multi_asset_parameters())
+def test_multi_asset_run(function, selected_model, input_data, output_keyword):
```

Comment:
I got an error in the translation and speech recognition:
![image](https://github.com/user-attachments/assets/43e4f3e0-27a9-44cd-a3a3-af4216f38bb9)
Running on prod env in aixplain-testing team.